requirements/base.in

```diff
@@ -8,31 +8,33 @@ python-decouple  # processing of envvar configs
 jsonschema
 
 # Framework libraries
-django < 3.0
+django==3.2
```

Comment:
Please remove all pinned versions from base.in file. We pin dependencies in base.txt